updateHostsFile.py

```diff
@@ -1126,6 +1129,60 @@ def remove_old_hosts_file(backup):
 # End File Logic
 
 
+def domain_to_idna(line):
+    """
+    Encode a domain which is presente into a line into `idna`. This way we
+    avoid the most encoding issue.
+
+    Parameters
+    ----------
+    line : str
+        The line we have to encode/decode.
+
+    Returns
+    -------
+    line : str
+        The line in a converted format.
+
+    Notes
+    -----
+    - This method/function encode only the domain to `idna` format because in
+        most cases the encoding issue is due to a domain which looks like
+        `b'\xc9\xa2oogle.com'.decode('idna')`.
```

Comment:
"This function encodes only" (it's a function)
"because in most cases, the" (add comma)

Reply:
Thanks @gfyoung